pkg/proto/configuration/global/global.proto

```diff
@@ -46,10 +53,20 @@ message PrometheusPushgatewayConfiguration {
   google.protobuf.Duration push_interval = 5;
 }
 
-message Configuration {
+message TracingConfiguration {
```

Comment:
Should this be called OpenCensusTracing, or would that be too explicit?

Reply:
Yeah, I think that's a bit specific to be getting at this point, I'd say people would probably want to reevaluate their tracing if we start supporting multiple tracing frameworks, which seems a bit... excessive anyhow.